samza-core/src/main/java/org/apache/samza/execution/ExecutionPlanner.java

```diff
@@ -211,9 +211,12 @@ private void validateConfig() {
           }
         }
       }
+
+      log.info("Inferred the partition count for the operator {} as {}.", join.getOpId(), partitions);
```

Comment:
Is there a place where we log all the inputs and their known partition counts before we start this traversal?

Reply:
We already log the partition count for all sources and sink in `updateExistingPartitions` which happens prior to traversal. Were you looking for something else or would that suffice?